### DISCUSSION TITLE
Recommendation for Focusrite on Linux questionable?

### DISCUSSION THREAD

**Discussion by blaueente** [jamulussoftware/jamulus#1587] [Technical Support & Help]
https://jamulus.io/kb/2021/01/05/Jamulus-Sound-Devices.html

While researching the Focusrite Scarlett range, I found complaints that newer devices (starting with 3rd gen) have removed hardware switches, and have those functions only usable with their driver software. However that software comes for Windows and Mac only, which makes the devices hard to use with Linux.

Can someone who actually uses that hardware confirm this?

**Comment by gilgongo** [comment]
Hi @blaueente - good question, although it may get more eyeballs in the Discussion (and on Facebook) rather than here. I'll move this to the former.

**Comment by jujudusud** [comment]
On LinuxMAO.org, some french owners said that it works as expected.

**Comment by trebmuh** [comment]
It depends on which model(s) we are talking about, the solo and 2i2 models works out of the box. The other might need some configuration to be done on a windows/macos before being usable on linux. That said, IIRC, there are some (free) software developed for those models. I don't know if there are already working fine or not.
